### DISCUSSION TITLE
Screen

### DISCUSSION THREAD

**Discussion by antoniomaze** [firemodels/fds#12657] [Announcements]
Hi,
I would like to model a screen that allows 50% of smoke to pass through it. Is the below model correct?

&INIT N_PARTICLES_PER_CELL=1, CELL_CENTERED=.TRUE., PART_ID='SCREEN', XB=192.0,192.0,202.0,213.0,3.0,10.0/

&PART ID='SCREEN', DRAG_LAW='SCREEN', FREE_AREA_FRACTION=0.5, STATIC=.TRUE., SURF_ID='SCREEN', ORIENTATION=1,0,0 /

&SURF ID='SCREEN', THICKNESS=0.00015, GEOMETRY='CYLINDRICAL', MATL_ID='.......' /

**Comment by rmcdermo** [comment]
Please use [Discussions](https://github.com/firemodels/fds/discussions) unless you are demonstrating a bug or requesting a feature.  I will convert and answer there.

**Comment by rmcdermo** [comment]
It is not really clear what you are asking.  Do you want to reduce the volume flow rate by 50 %?  Because if you have a duct with a screen and you flow smoke through it, then 100 % of the smoke will always go through the screen as long as there is any flow path at all, that is just mass conservation.  Reduction of volume flow rate depends on more than just the free area fraction, the drag goes as the velocity squared, so it depends also on the flow rate itself.
